docs/technical_specifications/ONBOARDING.md

```diff
@@ -35,19 +37,49 @@ behalf of that org for the registration of an entity, if not already present wit
             - check of the presence of kid claim
             - check that the certificate is not expired
             - check that the kid is unique and of the lenght of a minimum JWK thumbprint length
-    5. at least one administrative email contact (String)
+    5. TODO: at least one administrative email contact (String)
         1. validators:
             - Email Field
-    6. at least one administrative email contact (String)
-        1. validators:
-            - Email Field
-    7. URL where the RP page shows the SPID and CIE authentication button
+    6. URL where the RP page shows the SPID and CIE authentication button
         1. validators:
             - Null, just for demo purpose
-    8. SPID/CIE authentication request trigger url at the onboarding system testing OP
+    7. TODO: SPID/CIE authentication request trigger url at the onboarding system testing OP
         1. validators:
             - Null, just for demo purpose
 
+![Registration entity](../images/fillregistration.png)
+
+After successful submission it is redirected to the list of registered entities
+
+![entity list](../images/listentity.png)
+
+### Entity OnBoarding
+
+- all the required information are moved (copied) from the OnBoardin registration storage to the FederationDescendant storage
+- trust marks is automatically generated for the available profile, public or private
```

Comment:
not really true, the operator creates the profile, assigned to each descendant, by hands right now